lib/tr.js

```diff
@@ -17,8 +17,88 @@
 const utils = require('./utils.js');
 
 /**
+ * Callback returned by trs()
+ * @callback TranslationRequest~getTranslationRequestsCallback
+ * @param {Object} err -  error, or null
+ * @param {Object.<string, TranslationRequest>} trs - map from translation request ID to TranslationRequest
+ * Example: `{ 1dec633b: {…}}` if there was just one TR, id `1dec633b`
+ */
+
+/**
+ * Callback returned by getInfo and create
+ * @callback TranslationRequest~getTranslationRequestCallback
+ * @param {Object} err -  error, or null
+ * @param {TranslationRequest} tr - the returned TranslationRequest
+ */
+
+
+/**
+ * @typedef WordCountsInfo
+ * @type {object}
+ * @prop {string} sourceLanguage - bcp47 id of the source language, such as 'en'
+ * @prop {object.<string,number>} counts - map from target language to word count
+ */
+
+/**
+ * Possible status values for Translation Requests
+ * @enum {string}
+ * @prop {string} DRAFT - The translation request has not been submitted for processing. It may modified or cancelled.
+ * @prop {string} SUBMITTED - The translation request has been submitted for processing, but has not been accepted by the partner yet.
+ * @prop {string} STARTED - Work has started on the translation request.
+ * @prop {string} TRANSLATED - All work has been completed on the translation request. It has not been merged into the target resource data yet.
+ * @prop {string} MERGED - The translation results have been merged into the original resource bundles.
+ */
+const TranslationRequestStatus = {
+  // No values here, just @prop lines for documentation.
```

Comment:
see https://stackoverflow.com/questions/19093935/how-to-document-a-string-type-in-jsdoc-with-limited-possible-values